node/node.go

```diff
@@ -374,19 +383,50 @@ func (n *Node) initNetworking(primaryNetVdrs validators.Set) error {
 	return err
 }
 
+type NodeRuntimeState struct {
+	// The process id of the node
+	PID int
+	// URI to access the node API
+	// Format: [https|http]://[host]:[port]
+	URI string
+	// Address other nodes can use for bootstrapping
+	// Format: [host]:[port]
+	BootstrapAddress string
```

Comment:
I think we typically have all of our json use camelCase (with the exception of configs to align with the avalanchego config using `-`s.)